docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,338 @@
+
+Metrics definitions:
+
+**engaged_sessions** = count(case when engaged=True end)
+**engagement_rate** = engaged_sessions / count(distinct domain_sessionid)
+**conversion_count** =sum(cv__all_volume)
+**conversion_revenue** = sum(cv__all_total)
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_TA.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_TA.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_TA.png)
+
+</TabItem>
+
+</Tabs>
+
+### Entrance and Exit Points
+
+A separate overview is dedicated for the top 10 Entrance and Exit points with a breakdown on new and returning users.
+
+An `entrance point` is the first page URL within a session. Although the sessions table produces a field called first_page_url (first_page_url_scheme  || '://' || first_page_url_hist || first_page_url_path), for BI purposes a calculated field is being used on this dashboard to exclude GTM tags.
+
+Similarly an `exit point` is the last page URL within a session. It is calculated using last_page_url_scheme  || '://' || last_page_url_host || last_page_url_path.
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_EE.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_EE.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_EE.png)
+
+</TabItem>
+
+</Tabs>
+
+### Page Analytics
+
+The **Page Analytics** dashboard is aimed at giving insight into how the visitors interact with your web page.
+
+The `bounce rate` visualisation is essentially about the inverse of engagement rate, it shows the ratio of non-engaged sessions over the total sessions.
+
+There is also a pivot heat map called the `Entrance vs exit points` which show the top 10 most visited first pages based on total session_id count and displays the total sessions where the session ended (from the same top 10 pages only).
+
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_PA.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_PA.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_PA.png)
+
+</TabItem>
+
+</Tabs>
+
+## Customization
+
+The templates are designed with the intention that they can be used out of the box, therefore we refrained from adding too much styling that might not fit a wide audience's taste but there are plenty of things you can do, depending on the tool to make it your own.
+
+If you would like us to give you a more detailed guide on how you can do that you can request this and we could add a new Accelerator to cover this topic for you.
+
```

Comment:
At most I think maybe link here to the docs for each tool around themeing? Or even just their core docs site.

```suggestion

The templates are designed with the intention that they can be used out of the box, therefore we have only provided basic style and branding which we expect the user to further develop to make it their own.

```

Reply:
I realised I did not intend to phrase this around themes at all, that was just an example. Let me know if the rephrase works.